<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GammaRay::ModelInspectorWidget</class>
 <widget class="QWidget" name="GammaRay::ModelInspectorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>479</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QSplitter" name="mainSplitter">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="modelSplitter">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget_3">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Models</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="modelSearchLine"/>
        </item>
        <item>
         <widget class="GammaRay::DeferredTreeView" name="modelView">
          <property name="contextMenuPolicy">
           <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
          </property>
          <property name="uniformRowHeights">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Selection Models</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="GammaRay::DeferredTreeView" name="selectionModelsView">
          <property name="contextMenuPolicy">
           <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
          </property>
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <property name="uniformRowHeights">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Model Content</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="GammaRay::DeferredTreeView" name="modelContentView">
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectionBehavior::SelectItems</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Cell Content</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_8">
           <property name="minimumSize">
            <size>
             <width>55</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Model index:</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="indexLineEdit">
           <property name="minimumSize">
            <size>
             <width>72</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Invalid</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="minimumSize">
            <size>
             <width>55</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Internal id:</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="internalIdLineEdit">
           <property name="minimumSize">
            <size>
             <width>72</width>
             <height>0</height>
            </size>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_5">
           <property name="minimumSize">
            <size>
             <width>55</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Internal pointer:</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="internalPtrLineEdit">
           <property name="minimumSize">
            <size>
             <width>72</width>
             <height>0</height>
            </size>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_7">
           <property name="minimumSize">
            <size>
             <width>55</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Flags:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPlainTextEdit" name="flagsPlainTextEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>72</width>
             <height>0</height>
            </size>
           </property>
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
           </property>
           <property name="undoRedoEnabled">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="plainText">
            <string notr="true"/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="GammaRay::DeferredTreeView" name="modelCellView">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GammaRay::DeferredTreeView</class>
   <extends>QTreeView</extends>
   <header location="global">ui/deferredtreeview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
